client/source/js/modules/common/save-graph-as-directive.js

```diff
@@ -257,7 +292,39 @@ define(['angular', 'jquery', 'underscore', 'saveAs', './svg-to-png'],
               .error(function () {});
           };
 
-          initialize();
+          /**
+           * Exports the data of the graph in the format returned by the API
+           */
+          scope.exportMultiSheetFrom = function (graphs){
+            if(graphs.length == 0) { return scope.saySorry("Sorry, no graphs found");}
+            
+            var exportables = [];
+            var showAlert = false;
+            _(graphs).each(function (graph, index) {
+              var exportable = scope.getExportableFrom(graph)
+              if ( exportable )
+                exportables.push(exportable);
+              else
+                showAlert = true;
+            });
+            if(showAlert) { return scope.saySorry("Sorry, some graphs cannot be exported"); }
+
+            $http({url:'/api/project/exportall',
+                  method:'POST',
+                  data: exportables,
+                  headers: {'Content-type': 'application/json'},
+                  responseType:'arraybuffer'})
+              .success(function (response, status, headers, config) {
+                var blob = new Blob([response], { type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' });
```

Comment:
@Ashalynd looks like graphs are exported for those:
- AnalysisOptimization - template: optimization.html
- AnalysisScenarios - template: scenarios.html
- ModelCalibration - template: calibration.html
- ModelViewCalibration - template: cost-coverage.html